staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/subresource_test.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fieldmanager
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestFindEntries(t *testing.T) {
+	tests := []struct {
+		input    []metav1.ManagedFieldsEntry
+		path     []interface{}
+		expected []metav1.ManagedFieldsEntry
+	}{
+		{
+			input: []metav1.ManagedFieldsEntry{
+				{
+					Manager:    "test",
```

Comment:
you have the same manager twice here, I don't think that's valid for apply

Reply:
Makes sense, thanks for spotting!